### PR TITLE
Feature/ui/new menu

### PR DESCRIPTION
**Pull request by gcrsaldanha**
This PR implements the new menu with game difficulty choices.
- The difficulty choice is a variable at the UI instance (which is passed through the different "scenes": UI.game_difficulty)

Issue #83

### CODE REVIEW

File: chess/ui/ui.py [hunk 1]
```diff
@@ -27,6 +27,8 @@
 CELL_BORDER = 3
 IMAGES_FOLDER_PATH = 'chess/ui/assets/images'
 
+clock = pygame.time.Clock()
```
Comment (igormartire) [inline chess/ui/ui.py]: Acho que pode remover essa linha, n?

File: chess/ui/ui.py [hunk 2]
```diff
@@ -277,25 +285,131 @@ def promotion_callback(board):
...
+
+
 def menu(ui):
     print("menu")
     menu = True
+    difficulty = False
     quit = False
+
+    # region Main Menu
+    # TODO: create a class to encapsulate each menu option
```
Comment (igormartire) [inline chess/ui/ui.py]: Tirar TODO, ninguém vai fazer isso rs

Comment (gcrsaldanha) [inline chess/ui/ui.py]: hahahahaha :( 👍

File: chess/ui/ui.py (hunk 1 above)
Comment (gcrsaldanha) [inline chess/ui/ui.py]: Ops, ficou sem querer, valeu

**Pull request merged**